public_suffix_list.dat

```diff
@@ -11211,7 +11209,7 @@ zakopane.pl
 // Submitted by Ed Moore <Ed.Moore@lib.de.us>
 lib.de.us
 
-// 12CHARS: https://12chars.com
+// 12CHARS : https://12chars.com
```

Comment:
If you want this, could you please change psltool to actually lint for it? Otherwise it will just keep getting reintroduced.

Reply:
I'm not familiar with Go, so I cannot unfortunately. cc @danderson

Reply:
You are selling yourself short. Go is not that hard!

Reply:
I'll give it a shot, let's see how it goes.